src/main/java/fi/otavanopisto/kuntaapi/server/integrations/tpt/client/model/DocsEntry.java

```diff
@@ -0,0 +1,453 @@
+    this.tyonKestoTekstiYhdistetty = tyonKestoTekstiYhdistetty;
+  }
+
+  public String getId() {
+    return id;
+  }
+
+  public void setId(String id) {
+    this.id = id;
+  }
+
+  public Integer getHakuTyoaikakoodi() {
+    return hakuTyoaikakoodi;
+  }
+
+  public void setHakuTyoaikakoodi(Integer hakuTyoaikakoodi) {
+    this.hakuTyoaikakoodi = hakuTyoaikakoodi;
+  }
+
+  public Boolean getKesatyopaikka() {
+    return kesatyopaikka;
+  }
+
+  public void setKesatyopaikka(Boolean kesatyopaikka) {
+    this.kesatyopaikka = kesatyopaikka;
+  }
+
+  public Boolean getIlmoitettuTyomarkkinatLehdessa() {
+    return ilmoitettuTyomarkkinatLehdessa;
+  }
+
+  public void setIlmoitettuTyomarkkinatLehdessa(Boolean ilmoitettuTyomarkkinatLehdessa) {
+    this.ilmoitettuTyomarkkinatLehdessa = ilmoitettuTyomarkkinatLehdessa;
+  }
+
+  public Long getTimestamp() {
+    return timestamp;
+  }
+
+  public void setTimestamp(Long timestamp) {
+    this.timestamp = timestamp;
+  }
+
+  public String getTyoaika() {
+    return tyoaika;
+  }
+
+  public void setTyoaika(String tyoaika) {
+    this.tyoaika = tyoaika;
+  }
+
+  public Boolean getItsepalvelu() {
+    return itsepalvelu;
+  }
+
+  public void setItsepalvelu(Boolean itsepalvelu) {
+    this.itsepalvelu = itsepalvelu;
+  }
+
+  public Boolean getVahtipostitus() {
+    return vahtipostitus;
+  }
+
+  public void setVahtipostitus(Boolean vahtipostitus) {
+    this.vahtipostitus = vahtipostitus;
+  }
+
+  public String getHakemusOsoitetaan() {
+    return hakemusOsoitetaan;
+  }
+
+  public void setHakemusOsoitetaan(String hakemusOsoitetaan) {
+    this.hakemusOsoitetaan = hakemusOsoitetaan;
+  }
+
+  public Integer getPaikkalukumaara() {
+    return paikkalukumaara;
+  }
+
+  public void setPaikkalukumaara(Integer paikkalukumaara) {
+    this.paikkalukumaara = paikkalukumaara;
+  }
+
+  public String getKunta() {
+    return kunta;
+  }
+
+  public void setKunta(String kunta) {
+    this.kunta = kunta;
+  }
+
+  public String getTyonantajanNimi() {
+    return tyonantajanNimi;
+  }
+
+  public void setTyonantajanNimi(String tyonantajanNimi) {
+    this.tyonantajanNimi = tyonantajanNimi;
+  }
+
+  public Boolean getEuresTyopaikka() {
+    return euresTyopaikka;
+  }
+
+  public void setEuresTyopaikka(Boolean euresTyopaikka) {
+    this.euresTyopaikka = euresTyopaikka;
+  }
+
+  public String getYtunnus() {
+    return ytunnus;
+  }
+
+  public void setYtunnus(String ytunnus) {
+    this.ytunnus = ytunnus;
+  }
+
+  public Integer getMainAmmattikoodi() {
+    return mainAmmattikoodi;
+  }
+
+  public void setMainAmmattikoodi(Integer mainAmmattikoodi) {
+    this.mainAmmattikoodi = mainAmmattikoodi;
+  }
+
+  public Boolean getOppisopimus() {
+    return oppisopimus;
+  }
+
+  public void setOppisopimus(Boolean oppisopimus) {
+    this.oppisopimus = oppisopimus;
+  }
+
+  public Long get_version_() {
+    return _version_;
+  }
+
+  public void set_version_(Long _version_) {

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00100)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00117)